File: agent/test_db.py
import os
from supabase import create_client
from dotenv import load_dotenv
import asyncio

load_dotenv()
_supabase = create_client(os.environ["SUPABASE_URL"], os.environ["SUPABASE_SERVICE_KEY"])

async def test():
    try:
        res = _supabase.table("assignments").select("*").limit(1).execute()
        print("Assignments schema ok", res)
    except Exception as e:
        print("Error assignments:", e)
        
    try:
        res = _supabase.table("project_members").select("*").limit(1).execute()
        print("Project_members schema ok", res)
    except Exception as e:
        print("Error project_members:", e)

asyncio.run(test())

File: server/test_db.js
import 'dotenv/config'
import { createClient } from '@supabase/supabase-js'

const supabase = createClient(process.env.SUPABASE_URL, process.env.SUPABASE_SERVICE_KEY)

async function run() {
  const { data: ints } = await supabase.from('integrations').select('*').eq('provider', 'uazapi')
  console.log('--- Integrations ---')
  console.log(JSON.stringify(ints, null, 2))

  const { data: msgs } = await supabase.from('chat_messages').select('*').order('created_at', { ascending: false }).limit(5)
  console.log('--- Ultimas Mensagens ---')
  console.log(JSON.stringify(msgs, null, 2))
}

run()
